utils/apolloClient.js

```diff
@@ -0,0 +1,18 @@
+const { createHttpLink } = require('apollo-link-http');
```

Comment:
Please, rename the file to `apollo-client.js`

Reply:
fixed